R/attributes.R

```diff
@@ -349,10 +349,7 @@ graph.attributes <- function(graph) {
 #' @export
 "graph.attributes<-" <- function(graph, value) {
   ensure_igraph(graph)
-  if (!is.list(value) || (length(value) > 0 && is.null(names(value))) ||
```

Comment:
I don't see why value would have no length, but I added the case.

Reply:
well it does happen according to what happened in tests before I accounted for the case